po/de.po

```diff
@@ -6821,21 +6831,23 @@ msgstr "Aktuelle Framezeit und -nummer"
 msgid "Time of this frame relative to start and end of current subs"
 msgstr "Framezeit relativ zu Anfang und Ende der aktuellen Zeile"
 
+# ffms nutzt "Indexing"/"Indiziere"...
 #: ../src/video_provider_bestsource.cpp:127
 msgid "Scanning"
-msgstr ""
+msgstr "Suche"
 
+# ffms nutzt "Reading timecodes and frame/sample data" / "Lese Zeitcodes und Frame-/Sample-Daten"
 #: ../src/video_provider_bestsource.cpp:128
 msgid "Finding Keyframes and Timecodes..."
-msgstr ""
+msgstr "Finde Keyframes und Zeitcodes..."
 
 #: ../src/video_provider_vs.cpp:137
 msgid "Executing VapourSynth Script"
-msgstr ""
+msgstr "VapourSynth-Skript wird ausgeführt"
 
 #: ../src/video_provider_vs.cpp:147
 msgid "Failed to execute script! Press \"Close\" to continue."
-msgstr ""
+msgstr "Skriptausführung fehlgeschlagen! Drücke \"Schließen\" um fortzufahren."
```

Comment:
Should this be `Drücken Sie` to be consistent with previous translations? Though I can't say I know the convention for these kinds of phrases and I can't find a precedent in the existing .po file.